tests/channels/test_nbody_reactions.py

```diff
@@ -27,164 +25,138 @@ def reduce_violated_rules(
     [
         (
             (["p", "p~"], ["pi+", "pi0"]),
-            ["ChargeConservation", "isospin_conservation"],
-        ),
-        ((["eta"], ["gamma", "gamma"]), []),
-        ((["Sigma0"], ["Lambda", "pi0"]), ["MassConservation"]),
-        ((["Sigma-"], ["n", "pi-"]), []),
-        ((["e+", "e-"], ["mu+", "mu-"]), []),
-        # (  # https://github.com/ComPWA/expertsystem/issues/281
-        #     (["mu-"], ["e-", "nu(e)~"]),
-        #     ["MuonLNConservation", "SpinConservation"],
-        # ),  # https://github.com/ComPWA/expertsystem/issues/281
-        # (
-        #     (["mu-"], ["e-", "nu(e)"]),
-        #     [
-        #         "ElectronLNConservation",
-        #         "MuonLNConservation",
-        #         "SpinConservation",
-        #     ],
-        # ),
-        ((["Delta(1232)+"], ["p", "pi0"]), []),
-        ((["nu(e)~", "p"], ["n", "e+"]), []),
-        # (  # https://github.com/ComPWA/expertsystem/issues/281
-        #     (["e-", "p"], ["nu(e)", "pi0"]),
-        #     ["BaryonNumberConservation", "SpinConservation"],
-        # ),
-        ((["p", "p"], ["Sigma+", "n", "K~0", "pi+", "pi0"]), []),
+            {"ChargeConservation", "isospin_conservation"},
```

Comment:
Though not immediately relevant for this PR (the conversion from `ConservationRule` class to `str` was introduced earlier on), I now wonder if the strings should be made more concise/intuitive, like `violated_rules == {"isospin", "charge", "c_parity"}`. The naming distinction between function vs callable class is irrelevant for the output of this facade.

Reply:
Can of course be done, but some conversion code or `__str__` definitions

Reply:
I guess reliably only the latter works

Reply:
We can leave it like this, it would just be a service to the user, but the current output strings speak for themselves